hawkular-tenant-jaxrs-filter/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>hawkular-commons-parent</artifactId>
+    <groupId>org.hawkular.commons</groupId>
+    <version>0.7.3.Final-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hawkular-tenant-jaxrs-filter</artifactId>
+
+  <name>Hawkular REST Filter - Tenant</name>
+  <description>
+    A reusable implementation of a filter that makes Hawkular-Tenant required.
+    Provides also a @OptionalTenant so that endpoints can mark that they don't require a tenant.
```

Comment:
Nice catch! I renamed it to `@TenantRequired(boolean)` , which sounded better. I'll send a PR with the fix.